libs/tools/barista/src/builder/strapi.ts

```diff
@@ -76,12 +78,36 @@ export const strapiBuilder: BaPageBuilder = async (
         environment.strapiEndpoint,
       );
 
+  let categoriesData: BaStrapiCategory[] = [];
+
+  categoriesData = await fetchContentList<BaStrapiCategory>(
+    BaStrapiContentType.Categories,
+    { publicContent: isPublicBuild() },
+    environment.strapiEndpoint,
+  );
+
+  categoriesData = next
+    ? categoriesData.filter((data) => data.nextpages.length > 0)
+    : categoriesData.filter((data) => data.pages.length > 0);
+
   // Filter pages with draft set to null or false
   pagesData = pagesData.filter((page) => !page.draft);
 
+  // Results from transforming the pagecontent
   const transformed: BaPageBuildResult[] = [];
 
+  // Array of categories for corresponding design systems
+  const categories: string[] = [];
+
   for (const page of pagesData) {
+    /**
+     * Check if any pages have no a category assigned. These pages would be considered main pages.
+     * E.g. A page like the overview page describing the Design System itself does not relate to any category (angular-components or design-tokens for example).
+     * It is considered a main (or landing page) that the user should be able to navigate to.
+     */
```

Comment:
@rowa-audil Please use next time for comments inside the code the `//` annotation and not the jsDoc. JsDoc is used to annotate a method, property, etc but for code comments always use `//` as there is a difference on cleaning up the code.

```suggestion
    
     // Check if any pages have no a category assigned. These pages would be considered main pages.
     // E.g. A page like the overview page describing the Design System itself does not relate to any category (angular-components or design-tokens for example).
     // It is considered a main (or landing page) that the user should be able to navigate to.
```

Reply:
Ah right, thank you!